src/components/dropdown-group/dropdown-group.tsx

```diff
@@ -83,7 +83,7 @@ export class DropdownGroup {
       this.groupPosition > 0 ? <div class="dropdown-separator" role="separator" /> : null;
 
     return (
-      <Host role="menu">
+      <Host role="group">
```

Comment:
Could we add an `aria-label={groupTitle}` here as well? This would announce the title to screen reader users.